tasktiger/logging.py

```diff
@@ -11,6 +11,7 @@ def tasktiger_processor(logger, method_name, event_dict):
     if g["current_tasks"] is not None and not g["current_task_is_batch"]:
         current_task = g["current_tasks"][0]
         event_dict["task_id"] = current_task.id
+        event_dict["task_func"] = current_task.serialized_func
```

Comment:
Why not reuse the same `"func"` key as in the logs done by TT itself?

Reply:
I'm not comfortable introducing a global `func` label because it may clash with app logs.
It's not difficult to imagine that someone somewhere uses `logger.info("foo", func="...")`.

Reply:
Ideally I'd have all of these prefixed with `tasktiger`, but I don't want to mess with that too much ;-)